src/handlers/ipcHandlers/generalHandler.js

```diff
@@ -29,4 +30,16 @@ const getCurrentSeason = (e) => {
         console.error(error);
         e.sender.send("General:getCurrentSeasonOK", null);
     })
+}
+
+const getFormula = (e, formulaName) => {
+    let response = general.getFormula(formulaName);
+    console.log(response);
+    response.then(result => {
+        e.sender.send("General:getFormulaOK", result);
+    })
+    .catch(error => {
+        console.error(error);
+        e.sender.send("General:getFormulaOK", null);
```

Comment:
Why is it `getFormulaOK` but it's error and `null`?

Reply:
It's like saying "I'm fine" but you are not really fine :(

Reply:
:(



CAN WE CHANGE IT

Reply:
Hmm, other handlers did error handling by not returning anything to the frontend. Should I change it to that instead?

Reply:
Does a `General:getFormulaError` make sense

Reply:
Yeah we can easily just throw backend error that is catched on the fe later

Reply:
Then we'll note this when doing the revamp, resolving for now